src/MediaPlayer/MediaPlayer.cs

```diff
@@ -303,6 +303,7 @@ public enum MPErrorCode : nint {
 		NotSupported,
 		[iOS (10,1)]
 		Cancelled,
+		RequestTimedOut,
```

Comment:
Yes that's why I didn't add it.
Didn't want to remove the other one to not add noise.